crates/red_knot_test/README.md

```diff
@@ -25,6 +25,10 @@ match the resulting diagnostics with the assertions in the test. Assertions are
 Python comments. If all diagnostics and all assertions are matched, the test passes; otherwise, it
 fails.
 
+(If you are reading this document in raw Markdown source rather than rendered Markdown, note that
+the quadruple-backtick-fenced "markdown" language code block above is NOT itself part of the mdtest
+syntax, it's just how this README embeds an example mdtest Markdown document.)
```

Comment:
We could also link directly to https://github.com/astral-sh/ruff/blob/main/crates/red_knot_python_semantic/resources/mdtest/numbers.md, as an actual example of what the syntax should look like?

Reply:
ooh... We could also put this paragraph inside `<!---` and `--->`, so that you _only_ see the paragraph if you're reading the raw source, and the paragraph is hidden if you're reading a rendered version!

Reply:
> ooh... We could also put this paragraph inside `<!---` and `--->`, so that you _only_ see the paragraph if you're reading the raw source, and the paragraph is hidden if you're reading a rendered version!

Heh, I thought about this, but then thought it's perhaps too clever and just confusing. But maybe it's worth doing...

Reply:
Seems to work fine, let's do it :)

Reply:
Nice!! :D